lib/power-canvas.coffee

```diff
@@ -49,7 +49,7 @@ module.exports =
 
   getColorAtPosition: (left, top) ->
     offset = @editorElement.getBoundingClientRect()
-    el = atom.views.getView(@editor).shadowRoot.elementFromPoint(
+    el = (atom.views.getView(@editor).shadowRoot ? document).elementFromPoint(
```

Comment:
`elementFromPoint` is only defined on documents, per the spec (a shadow root is like a pseudodocument).